web/src/routes/MyPages/SurveyResultPage.tsx

```diff
@@ -1,12 +1,183 @@
+  height: 80vh;
+  background-color: ${(props) => props.theme.colors.container};
+`;
+
+const ListBox = styled.div`
+  z-index: 1;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border: ${(props) => props.theme.borderResultList};
+  border-radius: ${(props) => props.theme.borderRadius};
+  background-color: ${(props) => props.theme.colors.opposite};
+  box-shadow: 10px 10px 10px rgba(0, 0, 0, 0.4);
+  cursor: pointer;
+`;
+
+const ResultBox = styled.div`
+  height: 40vh;
+  width: 100%;
+  align-items: center;
+  border: ${(props) => props.theme.borderResultList};
+  border-radius: ${(props) => props.theme.borderRadius};
+  background-color: ${(props) => props.theme.colors.opposite};
+  box-shadow: 10px 10px 10px rgba(0, 0, 0, 0.4);
+  pointer-events: none;
+`;
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+`;
+
+const SurVeyResultPageTitle = styled.span`
+  text-align: left;
+  font-size: calc(2vh + 2vmin);
+  font-weight: 900;
+  margin-bottom: 2vh;
+  color: ${(props) => props.theme.colors.default};
+  cursor: pointer;
+`;
+
+const FontText = styled.span`
+  text-align: left;
+  font-size: calc(1vh + 1.4vmin);
+  font-weight: 900;
+  margin-left: 3vw;
+  min-width: 80px;
+  max-width: 20vw;
+  width: 140px;
+  color: ${(props) => props.theme.colors.default};
+`;
+
+export default function MyPage() {
+  const [theme, toggleTheme] = useTheme();
+  const [resultClickFirst, setResultClickFirst] = useState<boolean>(false);
+  const [resultClickSecond, setResultClickSecond] = useState<boolean>(false);
+  const navigate = useNavigate();
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+  };
+
+  const handleClick = (ClickNumber: number) => {
+    console.log(resultClickFirst);
+    if (ClickNumber === 1) {
+      setResultClickFirst(!resultClickFirst);
+    } else if (ClickNumber === 2) {
+      setResultClickSecond(!resultClickSecond);
+    }
+  };
 
-export default function SurveyResultPage() {
-  const { id } = useParams();
   return (
-    <div>
-      <h1>{`SurveyResultPage : ${id}`}</h1>
-    </div>
+    <Container theme={theme}>
+      <Header theme={theme} toggleTheme={toggleTheme} />
+      <SurveyResultContainer theme={theme}>
+        <Form onSubmit={handleSubmit}>
+          <SurVeyResultPageTitle style={{ marginBottom: '5vh' }} theme={theme} onClick={() => navigate('../mypage')}>
+            마이페이지 &gt; 설문 결과 조회
+          </SurVeyResultPageTitle>
+          <ListBoxContainer theme={theme}>
+            <ListBox
+              theme={theme}
+              onClick={() => handleClick(1)}
+              style={{
+                zIndex: resultClickFirst ? 1 : 'auto', // 설정 추가
+              }}
+            >
+              <FontText theme={theme}>test1</FontText>
+              <TwoArrow style={{ transform: resultClickFirst ? 'rotate(90deg)' : 'rotate(0deg)' }} />
+            </ListBox>
+            <ResultBox
+              style={{
```

Comment:
Please move this style codes to `const ResultBox = styled.div`. If you need props, then give it as parameter for `ResultBox`'s property or if you want to leave them, move another static styles to `ResultBox`.